### ISSUE TITLE
WPS for goodness of fit metrics

### ISSUE THREAD

**Issue by huard** [Ouranosinc/raven#19]

**Comment by huard** [comment]
See https://github.com/waderoberts123/Hydrostats

**Comment by richardarsenault** [comment]
This is good. For calibration we can use the ones in OSTRICH, but to evaluate the skill on other metrics we can use spotpy? Or do we not want to go there just yet?

**Comment by huard** [comment]
Makes sense. I'll let you define priorities, and don't a strong opinion about this.

**Comment by richardarsenault** [comment]
This WPS should be relatively straightforward: Input Qobs and Qsim, same length, and calculate the objective functions with SpotPy. Return a structure with the objective function values.
